src/expressions/DynamicContext.ts

```diff
@@ -83,6 +86,20 @@ class DynamicContext {
 		return this._temporalContext.implicitTimezone;
 	}
 
+	/**
+	 * Returns a pseudo-random integer and a decimal based on the passed seed
+	 */
+	public getRandomNumber(seed: number = this._defaultSeed): {currentInt: number, currentDecimal: number} {
+		// Seed a prng with the chosen seed. This is a linear congruential generator
+		const m = 134456;
```

Comment:
Is having 134456 different floats enough?

Additionally, does the spec define the range of random ints anywhere?

Reply:
I added a way better PRNG algorithm now, with a larder range.